src/tool/hpcrun/sample-sources/cuda.c

```diff
@@ -336,7 +336,7 @@ METHOD_FN(process_event_list, int lush_metrics)
     PAPI_event_code_to_name(self->evl.events[i].event, buffer);
     TMSG(CUDA, "metric for event %d = %s", i, buffer);
     int metric_id = /* weight */
-      hpcrun_set_new_metric_info_and_period(cuda_kind, strdup(buffer),
+      hpcrun_set_new_metric_info_and_period(cuda_kind, strdup(buffer),						//dejan: this is setting metric into hpcviewer column
```

Comment:
the comment should have your name elided. isn't this sample source dead code? I don't think this belongs in this commit.

Reply:
I am not sure how that ended up in this commit but I deleted it.